src/test/java/com/hivemq/mqtt/handler/connect/ConnectHandlerTest.java

```diff
@@ -149,11 +166,38 @@ public void setUp() throws Exception {
 
         when(channelPersistence.get(anyString())).thenReturn(null);
 
-        when(channelDependencies.getAuthInProgressMessageHandler()).thenReturn(
-                new AuthInProgressMessageHandler(mqttConnacker));
+        when(channelDependencies.getAuthInProgressMessageHandler()).thenReturn(new AuthInProgressMessageHandler(
+                mqttConnacker));
 
         defaultPermissions = new ModifiableDefaultPermissionsImpl();
 
+//        when(embeddedChannel.pipeline()).thenReturn(pipeline);
+//        when(ctx.channel()).thenReturn(embeddedChannel);
+//        when(ctx.pipeline()).thenReturn(pipeline);
+//        when(embeddedChannel.closeFuture()).thenReturn(channelFuture);
+//        when(embeddedChannel.attr(eq(ChannelAttributes.TAKEN_OVER))).thenReturn(new TestChannelAttribute<Boolean>(false));
+//        when(embeddedChannel.attr(eq(ChannelAttributes.AUTHENTICATED_OR_AUTHENTICATION_BYPASSED))).thenReturn(new TestChannelAttribute<Boolean>(true));
+//        when(embeddedChannel.attr(ChannelAttributes.QUEUE_SIZE_MAXIMUM)).thenReturn(new TestChannelAttribute<>(null));
```

Comment:
Can this be removed?

Reply:
its pretty essential, but ok, if you dont like it :)